cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -318,5 +318,18 @@ export default {
 
   checkIsItemCreated:(itemBarcode) => {
     cy.expect(Link(itemBarcode).exists());
+  },
+
+  checkMARCSourceAtNewPane() {
+    cy.do(actionsButton.click());
+    cy.expect([
+      Button({ id: 'edit-instance' }).exists,
```

Comment:
is it work?

I mean you pobably forgot () in exists. It will be surprised for me if it work without ()

Reply:
no, there was no error, but the method did not work either. I don't know how the brackets were removed)